### PR TITLE
Add Rituals Perfume Genie integration documentation

### PR DESCRIPTION
**Pull request by milanmeu**
## Proposed change

Add documentation for the Rituals Perfume Genie integration.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/46218
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/2243
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/rituals_perfume_genie.markdown [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ha_config_flow: true
+ha_release: 2021.3
+ha_category:
+  - Switch
+ha_codeowners:
+  - '@milanmeu'
+ha_domain: rituals_perfume_genie
+---
+
+The `Rituals Perfume Genie` integration allows you to control and monitor your Rituals perfume diffusers connected to your Rituals account. The integration is compatible with the first and second `2.0` version.
```
Comment (frenck) [inline source/_integrations/rituals_perfume_genie.markdown]:
```suggestion
The Rituals Perfume Genie integration allows you to control and monitor your Rituals perfume diffusers connected to your Rituals account. The integration is compatible with the first and second `2.0` version.
```

File: source/_integrations/rituals_perfume_genie.markdown [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+## Configuration
+
+To add the Rituals Perfume Genie integration to your installation:
+
+* Go to the Integrations menu: **Configuration** >> **Integrations** in the UI.
+* Click the button with the `+` sign at the bottom right of the screen.
+* Select **Rituals Perfume Genie** from the list of integrations.
+* Enter your Rituals credentials and click **SUBMIT**.
```
Comment (frenck) [inline source/_integrations/rituals_perfume_genie.markdown]:
```suggestion
To add the Rituals Perfume Genie integration to your installation:

- Go to the Integrations menu: **Configuration** >> **Integrations** in the UI.
- Click the button with the `+` sign at the bottom right of the screen.
- Select **Rituals Perfume Genie** from the list of integrations.
- Enter your Rituals credentials and click **SUBMIT**.
```

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

File: source/_integrations/rituals_perfume_genie.markdown [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+The Rituals Perfume Genie integration allows you to control and monitor your Rituals perfume diffusers connected to your Rituals account. The integration is compatible with the first and second `2.0` version.
+
+## Configuration
+
+To add the Rituals Perfume Genie integration to your installation:
+
+- Go to the Integrations menu: **Configuration** >> **Integrations** in the UI.
+- Click the button with the `+` sign at the bottom right of the screen.
+- Select **Rituals Perfume Genie** from the list of integrations.
+- Enter your Rituals credentials and click **SUBMIT**.
```
Comment (bdraco) [inline source/_integrations/rituals_perfume_genie.markdown]:
```suggestion
{% include integrations/config_flow.md %}
```

Will need to be rebased as well

**Review by frenck — CHANGES_REQUESTED** [review]
@milanmeu Seems like your rebase resulted in a merge? Please git rebase just your commits in order to get rid of all the unrelated commits in this PR.

**Comment by milanmeu** [comment]
I fixed the rebase. My commits doubled so I had to squash them.

File: source/_integrations/rituals_perfume_genie.markdown [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+ha_config_flow: true
+ha_release: 2021.3
+ha_category:
+  - Switch
+ha_codeowners:
+  - '@milanmeu'
+ha_domain: rituals_perfume_genie
+---
+
+The Rituals Perfume Genie integration allows you to control and monitor your Rituals perfume diffusers connected to your Rituals account. The integration is compatible with the first and second `2.0` version.
```
Comment (frenck) [inline source/_integrations/rituals_perfume_genie.markdown]:
```suggestion
The Rituals Perfume Genie integration allows you to control and monitor your Rituals perfume diffusers connected to your Rituals account. The integration is compatible with the first and second `2.0` versions.
```

**Review by frenck — APPROVED** [review]
Thanks, @milanmeu 👍

**Pull request merged**